js/ad_manager_vast.js

```diff
@@ -483,6 +546,85 @@ OO.Ads.manager(function(_, $) {
     this.initialize = function(amc) {
       this.amc = amc;
       this.amc.addPlayerListener(this.amc.EVENTS.INITIAL_PLAY_REQUESTED, _.bind(this.initialPlay, this));
+      this.amc.addPlayerListener(this.amc.EVENTS.FULLSCREEN_CHANGED, _.bind(_onFullscreenChanged, this));
+      this.amc.addPlayerListener(this.amc.EVENTS.SIZE_CHANGED, _onSizeChanged);
+    };
+
+    /**
+     * Callback when the media file is loaded. Once is loaded we can initialize the ad
+     * @private
+     */
+    var _initializeAd = _.bind(function() {
```

Comment:
If this function is used for VPAID only, let's make note of this in the jsdoc and function name. With the influx of new VPAID functions, we should try to make these distinctions where appropriate. Maybe even encapsulate them into a separate object? Though that would be a talk for a later time.